ccd-definition/ComplexTypes/2_ChildParty.json

```diff
@@ -126,7 +126,7 @@
     "FieldType": "AddressUK",
     "ElementLabel": "Current address",
     "SecurityClassification": "Public",
-    "FieldShowCondition": "livingSituation=\"*\""
+    "FieldShowCondition": "livingSituation=\"*\" OR showAddressInConfidentialTab=\"Yes\""
```

Comment:
why do we need this showAddressInConfidentialTab? Unless I am missing something. It seems when I remove it  from this json file and from ChildParty.Java everything still works as expected?

Reply:
because address only shows when living situation has a value. When we build the confidential version, this is set to null. I don't want to mess around with living situation show hide in confidentiality, so I added the new field showAddressInConfidentialTab. This then triggers the show condition in the tabs :)

Reply:
Oh ok cool get you now thanks for the explanation :)